priv/site/_includes/info.slime

```diff
@@ -17,20 +17,11 @@
       | This event is for you. Brew some magic!
 
   .rectangles aria-hidden="true"
-    div
-      video.lozad autoplay=true muted=true loop=true data-poster="<%= url_for("videos/schedule.jpg") %>"
-        source data-src="<%= url_for("videos/schedule.webm") %>" type="video/webm"
-        source data-src="<%= url_for("videos/schedule.mp4") %>" type="video/mp4"
-
-    div
-      video.lozad autoplay=true muted=true loop=true data-poster="<%= url_for("videos/tickets.jpg") %>"
-        source data-src="<%= url_for("videos/tickets.webm") %>" type="video/webm"
-        source data-src="<%= url_for("videos/tickets.mp4") %>" type="video/mp4"
-
-    div
-      video.lozad autoplay=true muted=true loop=true data-poster="<%= url_for("videos/info.jpg") %>"
-        source data-src="<%= url_for("videos/info.webm") %>" type="video/webm"
-        source data-src="<%= url_for("videos/info.mp4") %>" type="video/mp4"
+    = for info <- ["schedule", "tickets", "info" ] do
+      div
+        video.lozad autoplay=true muted=true loop=true data-poster=url_for("videos/#{info}.jpg")
+          source data-src=url_for("videos/#{info}.webm") type="video/webm"
+          source data-src=url_for("videos/#{info}.mp4") type="video/mp4"
```

Comment:
Can we use something like `source` (?) as a keyword instead? `info` seems a bit odd. Other than that, nicely done 👍

Reply:
Yes, that sounds much better. Changed to `source`.